src/other/Html.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["d3", "../common/HTMLWidget"], factory);
+    } else {
+        root.other_Html = factory(root.d3, root.common_HTMLWidget);
+    }
+}(this, function (d3, HTMLWidget) {
+    function HTML() {
+        HTMLWidget.call(this);
+        this._tag = "div";
+        this._drawStartPos = "origin";//"center";
+    }
+    HTML.prototype = Object.create(HTMLWidget.prototype);
+    HTML.prototype.constructor = HTML;
+    HTML.prototype._class += " other_Html";
+    
+    HTML.prototype.publish("html", "", "string", "Html to render", null, { tags: ["Basic"] });
+    HTML.prototype.publish("overflowX", "auto", "set", "Html container overflow-x behavior", ["auto","scroll","visible","hidden","inherit","initial","overlay"], { tags: ["Basic"] });
+    HTML.prototype.publish("overflowY", "scroll", "set", "Html container overflow-y behavior", ["auto","scroll","visible","hidden","inherit","initial","overlay"], { tags: ["Basic"] });
```

Comment:
Not necessarily (I do this a lot) - in the users css they would typically do this:
```
#myDiv .other_Html div {
    ...
}
```
And then it will take precedence...